spec/models/manageiq/providers/vmware/infra_manager/provision_workflow_spec.rb

```diff
@@ -147,9 +147,9 @@
     end
 
     context 'network selection' do
-      let(:s11) { FactoryGirl.create(:switch, :name => "A") }
-      let(:s12) { FactoryGirl.create(:switch, :name => "B") }
-      let(:s13) { FactoryGirl.create(:switch, :name => "C") }
+      let(:s11) { FactoryGirl.create(:switch, :name => "A", :shared => false) }
+      let(:s12) { FactoryGirl.create(:switch, :name => "B", :shared => false) }
+      let(:s13) { FactoryGirl.create(:switch, :name => "C", :shared => false) }
```

Comment:
Should probably leave shared `nil` in the tests since that matches what we'll see on a real system.